pyscriptjs/src/main.ts

```diff
@@ -168,10 +171,21 @@ export class PyScriptApp {
     async afterRuntimeLoad(runtime: Runtime): Promise<void> {
         console.assert(this.config !== undefined);
 
+        this.fetchPaths(runtime);
+
         this.logStatus('Python startup...');
         await runtime.loadInterpreter();
         this.logStatus('Python ready!');
 
+        this.logStatus(`files already fetched, but yet to be copied: ${this._filesToCopy}`);
```

Comment:
This is the line that eslint is complaining about. It complains because you are passing the whole `filesToCopy` which is `{ path: string; content: Uint8Array; }`, we probably want to change this to use just the path right?

```suggestion
        this.logStatus(`files already fetched, but yet to be copied: ${this._filesToCopy.path}`);
```

Reply:
Aah thanks, I thought the declaration had an issue. Might have missed the line number.
In any case, accessing `.path` won't work since it's an array.